src/Engine/System/RenderSystem.hpp

```diff
@@ -8,44 +8,65 @@
 #include <string>
 #include <type_traits>
 #include <typeindex>
+#include <iostream>
 
 #include "System.hpp"
 
 #include "Shaders/Shader.hpp"
 #include "Shaders/DiffuseShader.hpp"
 #include "Shaders/BounderShader.hpp"
+#include "Component/RenderComponents/DiffuseRenderComponent.hpp"
+
 
-#include <iostream>
 
-class RenderSystem : public System {
+// static class
+class RenderSystem {
 
     friend Scene;
 
-    private: // only scene can create system
+    public:
+
+    static constexpr SystemID ID = SystemID::render;
 
-    RenderSystem(std::vector<Component *> & components);
+    static constexpr float k_defNear = 0.01f, k_defFar = 500.0f;
```

Comment:
Don't like these here. What are they used for? Why not pull them from a camera?

Reply:
I had move near/far out of camera to be render system dependent. I've moved it back now